### DISCUSSION TITLE
Develop bundles for the first release [6/1]

### DISCUSSION THREAD

**Discussion by Nic-Ma** [Project-MONAI/MONAI#4219] [General]
According to the discussion of our internal meeting, would be nice to develop several bundles for the first release [6/1]:

- [x] `Spleen` segmentation with UNet - @Nic-Ma 
- [x] `DeepEdit` of MONAI label - @diazandr3s 
- [x] `Swin UNETR` segmentation pipeline - @ahatamiz 
- [x] `DiNTS` AutoML pipeline - @dongyang0122 
- [ ] `Detection` pipeline - @Can-Zhao 
- [x]  `Pathology` tumor detection - @drbeh 
- [x] `Brats` segmentation with SegResNet - @yiheng-wang-nv 
- [ ] Example to show `bundle + NVFlare` - @yiheng-wang-nv @YuanTingHsieh 
- [x] Example to show `bundle + auto-deploy` - @ericspod

**Comment by yiheng-wang-nv** [comment]
Updates:

Nic and I prepared the proposal of how to contribute a bundle (https://github.com/Project-MONAI/MONAI/wiki/Proposal-of-MONAI-Model-Zoo-Contribution-Pipeline).

I will create the bundle of `Brats segmentation with SegResNet`, and test the whole process soon.

**Comment by wyli** [comment]
(about the deployment of bundles https://github.com/Project-MONAI/MONAI/discussions/4237)

**Comment by ericspod** [comment]
@aylward Model Zoo discussion is here with list of bundles to prepare and zoo proposal.

**Comment by diazandr3s** [comment]
Bundle for DeepEdit is here: https://github.com/Project-MONAI/tutorials/pull/694

Waiting for the Model Zoo repo to create the PR :)

**Reply by yiheng-wang-nv** [reply]
Thanks @diazandr3s for the quick updates. Will let you know after the repo is open asap

**Comment by Nic-Ma** [comment]
Hi @ericspod ,

Could you please help share some updates and next steps for the `auto-deploy` part?

Thanks in advance.

**Reply by ericspod** [reply]
I'm still working on finalising the operator for the App SDK, current work is [here](https://github.com/Project-MONAI/monai-deploy-app-sdk/tree/bundle_addition). It's using notebooks that create basic MedNIST and Speen segmentation bundles that  we can also include in the zoo later.

**Reply by ericspod** [reply]
@MMelQin is currently spearheading the effort [here](https://github.com/Project-MONAI/monai-deploy-app-sdk/tree/mqin/Issue286_Inference_Operator_for_MONAI_Bundle).

**Reply by Nic-Ma** [reply]
Hi @ericspod ,

Is there a PR or something there? I want to get the overall understanding first.

Thanks.

**Reply by ericspod** [reply]
There will be a PR shortly. The key class is [here](https://github.com/Project-MONAI/monai-deploy-app-sdk/blob/mqin/Issue286_Inference_Operator_for_MONAI_Bundle/monai/deploy/operators/monai_bundle_inference_operator.py) which implements an inference operator initialized with information from the bundle.

**Comment by ericspod** [comment]
Bundle for the MedNIST GAN example is up: https://github.com/Project-MONAI/model-zoo/pull/31

**Comment by ericspod** [comment]
Bundle for 2D cardiac segmentation network: https://github.com/Project-MONAI/model-zoo/pull/32

**Reply by MMelQin** [reply]
I'll be pushing the changes that integrated Eric contribution today to the branch and create a PR with an example supporting the Spleen bundle from Model Zoo.

**Comment by ericspod** [comment]
@MMelQin has put together [this notebook](https://github.com/Project-MONAI/monai-deploy-app-sdk/blob/main/notebooks/tutorials/06_monai_bundle_app.ipynb) demonstrating the use of MONAI Deploy with bundles through its bundle operator. This uses existing bundles so should tick the box for the item on deploy.

**Reply by Nic-Ma** [reply]
Glad to know that, cool!
Thanks for the sharing.

**Comment by bhashemian** [comment]
Pathology tumor detection bundle is merged.

**Comment by vikashg** [comment]
@Nic-Ma  We have decided to build and share a Breast Density Classifier from Mayo Clinic. I am working on it,

**Reply by Nic-Ma** [reply]
Cool! Glad to see your contribution!

Thanks.
